gcs/CONFIGURATION.md

```diff
@@ -152,6 +152,19 @@ provider.
 
     The implementation of the AccessTokenProvider interface used for GCS
     Connector.
+    
+*   `fs.gs.should.authenticate.per.request` (default: `false`)
```

Comment:
Totally agree, the name can use more thought. According to the below comments, this will be merged with the other flag to get something like "auth.token.method"